data/newsletter/issue-374.markdown

```diff
@@ -0,0 +1,75 @@
+
+- [Haskell: Sourced Recursion Schemes](https://apotheca.io/articles/Sourced-Recursion-Schemes.html) by Leo Dillinger
+  > This is the third article in a series about recursion, focused on developing the birecursion-schemes library, although it is starting to grow beyond that scope.
+
+- [ICFP Pearl preprint on rec-def](https://www.joachim-breitner.de/blog/805-ICFP_Pearl_preprint_on_rec-def) by Joachim Breitner
+  > It is about the idea of using Haskell's inherent ability to define recursive equations, and use them for more than just functions and lazy data structures.
+
+- [LTS 21 release for ghc-9.4 and Nightly now on ghc-9.6](https://www.stackage.org/blog/2023/06/announce-lts-21-nightly-ghc9.6) by Jens Petersen & Andreas Ländle
+  > The Stackage team is very happy to announce the first Stackage LTS version 21 snapshot has been released this week, based on GHC stable version 9.4.5.
+
+- [Monadic lenses are the optic for right monad modules III](https://julesh.com/2023/06/28/monadic-lenses-are-the-optic-for-right-monad-modules-iii/) by Jules Hedges
+  > If we'd like both our forwards and backwards passes to live in our monad then this is the simplest case that there is no possible concrete description, without using any quantification.
+
+- [r/haskell will remain read-only](https://np.reddit.com/r/haskell/comments/14kb9y7/rhaskell_will_remain_readonly/) by Taylor Fausak
+  > Until further notice, r/haskell will be read-only. You can still comment, but you cannot post.
+
+- [Writing prettier Haskell with Unicode Syntax and Vim](https://alt-romes.github.io/posts/2023-06-21-haskell-unicode-syntax-vim.html) by Rodrigo Mesquita
+  > A short write-up on combining digraphs, a feature built-in to vim, and Haskell's `UnicodeSyntax` extension, to easily write beautiful Haskell programs with unicode symbols.
+
+## Jobs
+
+- [PhD Position at KU Leuven](https://www.kuleuven.be/personeel/jobsite/jobs/60229600)
+
+Trying to hire a Haskell developer?
+You should [advertise with us](https://haskellweekly.news/advertising.html)!
+
+## In brief
+
+- [Episode 29: ZuriHac](https://haskell.foundation/podcast/29/) by The Haskell Interlude
+  > Joachim Breitner went to ZuriHac 2023 in order to bring the spirit of the biggest Haskell community event to you.
+
+- [Haskell Foundation DevOps Weekly Log, 2023-06-28](https://discourse.haskell.org/t/haskell-foundation-devops-weekly-log-2023-06-28/6736?u=taylorfausak) by Bryan Richter
+  > After coming back from a trip to see the solstice midnight sun, I continued the rewrite of spuriobot (in progress).
+
+- [The Haskell Unfolder Episode 6](https://discourse.haskell.org/t/the-haskell-unfolder-episode-6-computing-type-class-dictionaries/6727?u=taylorfausak) by Andres Löh
+  > A function with a `Show a` constraint wants evidence that type `a` has a `Show` instance. But what if we want to return such evidence from a function?
+
+- [ZuriHac 2023 videos](https://www.youtube.com/playlist?list=PLOvRW_utVPVmzDGGOJ2amgVBK168Vemke)
+
```

Comment:
Could you please add a video series, Learn Haskell by Building a blog generator? 

```suggestion

- [Learn Haskell by Building a blog generator](https://www.youtube.com/watch?v=ZL0qExCnO8g&list=PLxn_Aq3QlOQcXoHWdzxnnuGlGWNXJg43R&ab_channel=ImpurePics)
  > In this video series, we implement a simple static blog generator in Haskell, converting documents written in a custom markup language to HTML.

```

Reply:
Of course! Thanks for the suggestion.